content/design-systems-for-developers/react/pt/distribute.md

```diff
@@ -0,0 +1,384 @@
+---
+title: 'Distribuir o IU numa organização'
+tocTitle: 'Distribuição'
+description: 'Aprenda como empacotar e importar o seu sistema de design para outras aplicações'
+commit: 3a5cd35
+---
+
+De uma perspetiva arquitetónica, os sistemas de design são apenas mais uma dependência de frontend. Não são tão diferentes de outras dependências populares, tais como moment ou lodash. Os componentes de IU são apenas código, como tal podemos utilizar técnicas estabelecidas para reutilização de código.
+
+Este capítulo ilustra como distribuir sistemas de design, desde empacotamento dos componentes de IU até à importação dos mesmos noutras aplicações. Iremos desvendar técnicas que economizam tempo em termos de publicação e versões.
+
+![Propagar componentse em sites](/design-systems-for-developers/design-system-propagation.png)
+
+## Empacotar o sistema de design
+
+As organizações possuem milhares de componentes de IU espalhados por inúmersas aplicações. Anteriormente extraímos os componentes comuns para o sistema de design. Agora é necessário reintroduzi-los de volta nas aplicações
+
+O nosso sistema de design usa o gestor de pacotes npm para Javascript, para lidar com a gestão de dependências, distribuição e controlo de versões. 
+
+Existem muitas outros métodos válidos para empacotar os sistemas de design. Dê uma olhadela aos sistemas de design da Lonely Planet, Auth0, Salesforce, GitHub e Microsoft para ver a diversidade de abordagens. Alguns lançam cada componente como um pacote separado. Outros lançam todos os componentes num só pacote.
+
+Para sistemas de design recém-nascido, a forma mais direta é a publicação de um pacote com controlo de versões que encapsula:
+
+- 🏗 Componentes interface utilizador comuns
+- 🎨 Tokens de design(também conhecidos como variáveis de estilo)
+- 📕 Documentação
+
+![Package a design system](/design-systems-for-developers/design-system-package.jpg)
+
+## Prepare o seu sistema de design para ser exportado
+
+Utilizámos como ponto de partida para o nosso sistema de design o create-react-app, com isto ainda existem alguns vestigios da aplicação inicial criada pelo pacote. Vamos fazer agora uma limpeza.
+
+Primeiro deveremos adicionar um ficheiro README.md básico:
+
+```markdown
```

Comment:
let me take a look at it. Probably it might have slipped through.

Reply:
I've checked it and the markdown associated, is being left in it's original form, since i'm treating it as it was a block of code and it's costumary to leave it as is. I forgot to mention this block in the pr description. @domyen and @tmeasday i'm pinging both of you, as i don't know who's responsible for this section. Do you guys mind if i translate the markdown associated  to "Prepare your design system for export" in the "distribute" section of the guide? If you guys don't mind i'll adjust and commit the changes. I can relate to staying in the orginal form as the package is technically going to npm and with that the readme should stay in english for a "universal" understanding. But i can also relate to @joaogarin's input as the said readme would be custom tailored to the reader.

Reply:
Hey @jonniebigodes thanks for asking. I think we should leave it in English because the corresponding code commit in the example repo is also English. I don't want to have multiple example repos/branches for different languages (tough to maintain).

Reply:
Fine by me! I'll leave it as is.